src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -309,6 +327,13 @@ Terminal::UpdateSelectionParams Terminal::ConvertKeyEventToUpdateSelectionParams
     return std::nullopt;
 }
 
+bool Terminal::MovingStart() const noexcept
+{
+    // true --> we're moving start endpoint ("higher")
+    // false --> we're moving end endpoint ("lower")
+    return _selection->start == _selection->pivot ? false : true;
+}

```

Comment:
```suggestion
bool Terminal::MovingEnd() const noexcept
{
    // true --> we're moving end endpoint ("lower")
    // false --> we're moving start endpoint ("higher")
    return _selection->start == _selection->pivot;
}
```
This is just plain simpler. Then I can reuse it in `UpdateSelection()` too.